tests/features/poetry_translations.feature

```diff
@@ -157,3 +157,90 @@ Feature: Poetry translations
+    And I reload the page
+    Then I should see "Ongoing in Poetry" in the "Bulgarian" row
+    And I should see "Ongoing in Poetry" in the "Czech" row
+
+    # Cancel a language and leave other ongoing
+    When Poetry updates the status for "My title to translate" as "Ongoing" with the following individual statuses
+      | language  | status    |
+      | Bulgarian | Ongoing   |
+      | Czech     | Cancelled |
+    And I reload the page
+    Then I should see "Cancelled in Poetry" in the "Czech" row
+    And I should see "Ongoing in Poetry" in the "Bulgarian" row
+
+    # Make an update request
+    When I select the languages "German" in the language list
+    And I press "Request a DGT translation update for the selected languages"
+    And I fill in "Requested delivery date" with "05/04/2050"
+    And I fill in the the "first" "Author" field with "john"
+    And I fill in "Secretary" with "john"
+    And I fill in "Contact" with "john"
+    And I fill in the the "first" "Responsible" field with "john"
+    And I fill in the the "second" "Responsible" field with "john"
+    And I fill in the the "second" "Author" field with "john"
+    And I fill in "Requester" with "john"
+    And I press "Send request"
+    Then I should see "Submitted to Poetry" in the "Bulgarian" row
+    And I should see "Submitted to Poetry" in the "German" row
+    And I should not see "Cancelled in Poetry" in the "Czech" row
+
+    # The translations get accepted in Poetry
+    When the Poetry translation request of "My title to translate" in "Bulgarian, German" gets accepted
+    And I reload the page
+    Then I should see "Ongoing in Poetry" in the "Bulgarian" row
+    And I should see "Ongoing in Poetry" in the "German" row
+
+    # Cancel the request
+    When Poetry updates the status for "My title to translate" as "Cancelled" with the following individual statuses
+      | language  | status    |
+      | Bulgarian | Ongoing   |
+      | German    | Cancelled |
+    And I reload the page
+    Then I should see "Cancelled in Poetry" in the "Bulgarian" row
+    And I should see "Cancelled in Poetry" in the "German" row
+
+    # Send a new translation request
+    When I select the languages "Spanish" in the language list
+    And I press "Request a DGT translation for the selected languages"
+    And I fill in "Requested delivery date" with "05/04/2050"
+    And I fill in the the "first" "Author" field with "john"
+    And I fill in "Secretary" with "john"
+    And I fill in "Contact" with "john"
+    And I fill in the the "first" "Responsible" field with "john"
+    And I fill in the the "second" "Responsible" field with "john"
+    And I fill in the the "second" "Author" field with "john"
+    And I fill in "Requester" with "john"
+    And I press "Send request"
+    Then I should see "The request has been sent to DGT."
+    And I should not see "Cancelled in Poetry" in the "Bulgarian" row
+    And I should not see "Cancelled in Poetry" in the "Czech" row
```

Comment:
For completeness sake we should probably also add:
And I should see "Submitted to Poetry" in the "Spanish" row